src/App.js

```diff
@@ -1,12 +1,12 @@
 import logo from './logo.svg';
 import './App.css';
-import workList from './components/workList';
+import WorkList from './components/workList.js'
 
 function App() {
   return (
     <div className="App">
       <img src={logo} className="App-logo" alt="logo" />
-      <workList />
+      <WorkList />
```

Comment:
Again, notice the capitalized component name. I don't know exactly why, but it wouldn't render without it!